### ISSUE TITLE
v9alpha Less Compile Exception

### ISSUE THREAD

**Issue by JimMichael** [SparkDevNetwork/Rock#3789]
### Description

This may be something specific to my system, but I'm reporting it here in case it's not (and I also have no idea why it's occurring). Whenever my test instance of v9alpha2 is restarted, I get the following exceptions:
![image](https://user-images.githubusercontent.com/3903408/61078129-9a1de700-a3e5-11e9-9a20-a366256f4245.png)

The strange thing is that this instance started out as an exact copy of prod (RockRMS directory and all) and was then upgraded to v9, and I see no such exceptions in prod. I also see about 20 `_css-overrides.less` files in various theme directories so I have no idea which one it thinks is "missing."

### Versions

* **Rock Version:** v9 alpha2

**Comment by garrettjohnson** [comment]
@JimMichael Can you try compiling your themes manually and finding the theme that won't compile?

**Comment by JimMichael** [comment]
Looks like Dashboard Stark and Kiosk Stark are the culprits. Looks like they don't compile on 8.8, either (verified on demo site) but DO compile fine on prealpha (v10).

**Comment by nairdo** [comment]
Garrett and I tracked the issue down. A few years ago, the Rock update builder tool was changed to purposely exclude "_variable-overrides.less" and "_css-overrides.less" from packages.  _(I'm changing it now to only excluded them on CHANGED, not on ADDED.)_

This means the DashboardStark/Styles/_css-overrides.less and the KioskStark/Styles/_css-overrides.less added on Nov 2018 (https://github.com/SparkDevNetwork/Rock/commit/6b99ed55e16ab45fda28588e49a80ddf4c959c8e) never made it into the 8.5 release package.  I confirmed this.

We're fixing this by manually including them into the 9.0 package.  This means the 9.0 update will wipe out those files if someone has already created local overrides on their Rock system for those themes (which they should _not_ have since the stark themes are meant as a copy-paste starting point).
